awx/ui/src/screens/InstanceGroup/InstanceDetails/InstanceDetails.js

```diff
@@ -275,10 +275,13 @@ function InstanceDetails({ setBreadcrumb, instanceGroup }) {
           </Tooltip>
           {me.is_superuser && instance.node_type !== 'control' && (
             <DisassociateButton
-              verifyCannotDisassociate={!me.is_superuser}
+              verifyCannotDisassociate={
+                !me.is_superuser || instanceGroup.name === 'controlplane'
```

Comment:
From line 276, we just show this button if it is a super_user. We could simplify.
```suggestion
               instanceGroup.name === 'controlplane'
```